fontforge/splinefont.h

```diff
@@ -127,19 +127,22 @@ typedef struct ibounds {
 
 
 enum val_type { v_int, v_real, v_str, v_unicode, v_lval, v_arr, v_arrfree,
-		v_int32pt, v_int16pt, v_int8pt, v_void };
+		v_void };
+
+enum val_flags {
+    vf_none = 0,
+    vf_dontfree = (1 << 1)
```

Comment:
Is this interacting with another set of flags, or are we just reserving the low bit?

Reply:
There's no interaction with other flags, so this is the only flag in this set at the moment.

I was thinking about adding another flag to get rid of the `vf_arrayfree` type, but I left that out for now, since that required more thought than I'm bothered investing in right now.